src/main/java/org/jenkinsci/plugins/scriptsecurity/sandbox/groovy/SandboxInterceptor.java

```diff
@@ -163,6 +163,14 @@
         if (m != null && whitelist.permitsMethod(m, receiver, args0)) {
             return super.onGetProperty(invoker, receiver, property);
         }
+        if (m == null) {
+            String booleanGetter = "is" + Functions.capitalize(property);
+            m = GroovyCallSiteSelector.method(receiver, booleanGetter, args0);
```

Comment:
Unfortunately the way these methods work, you cannot “reuse” a local variable referring to candidate members without misreporting the `RejectedAccessException` in corner cases. Hence the long list of arguments to `rejectField`. I really need to refactor this stuff to make it easier to add clauses; probably need a helper method that takes a closure of some kind.